src/common.js

```diff
@@ -0,0 +1,222 @@
+import Todo from './todo';
+
+const addTodoFrm = (project, fun) => {
+  const todoScreen = document.getElementById('todoScreen');
+  const form = document.createElement('form');
+  const title = document.createElement('input');
+  const des = document.createElement('input');
+  const dueDate = document.createElement('input');
+  const priority = document.createElement('input');
+  const btn = document.createElement('button');
+
+  btn.innerHTML = 'Create';
+  btn.setAttribute('type', 'button');
+
+  title.setAttribute('type', 'text');
+  title.setAttribute('placeholder', 'Title');
+
+  des.setAttribute('type', 'text');
+  des.setAttribute('placeholder', 'Description');
+
+  dueDate.setAttribute('type', 'date');
+  dueDate.setAttribute('placeholder', 'Due Date');
+
+  priority.setAttribute('type', 'number');
```

Comment:
Opps, your priority should not be a number.

It should be either of the following; low, medium, or high

![image](https://user-images.githubusercontent.com/31897434/128724634-f17cd423-3d01-4533-be10-33125ecf7153.png)

Reply:
✔️